app/helpers/dashboard_helper.rb

```diff
@@ -90,4 +90,14 @@ def report_color
     }
   end
 
+  def auto_refresh_button(options = {})
+    on = options[:defaults_to] ? "on" : "off"
+    if params[:auto].present?
+      on = params[:auto] == "0" ? "off" : "on"
+    end
+    tooltip = _("Auto refresh #{on}")
```

Comment:
Write out both `_('Auto refresh on')` and `_('Auto refresh off')` rather than interpolating the word on/off, and use the appropriate string in the link_to call.

The problem with interpolating is that the translation may be different depending on the word being interpolated, so you need to spell the variations out.